src/benchmark.cc

```diff
@@ -906,6 +895,13 @@ void ParseCommandLineFlags(int* argc, const char** argv) {
   }
 }
 
+Benchmark* RegisterBenchmarkInternal(Benchmark* bench) {
```

Comment:
ack. This can't take unique_ptr directly to satisfy my need that no c++ library types cross the library boundary.